logging/unit_tests/test_client.py

```diff
@@ -562,6 +629,9 @@ def create_scoped(self, scope):
         self._scopes = scope
         return self
 
+    def authorize(self, _):
```

Comment:
This is not an adequate solution to the "Pylint is angry" problem. ISTM there are two options:

- keep track of what has been authorized, e.g. `self._authorized = http` and then check that value later, if you like?
- Switch to using `mock` (we recently starting using it, Hallelujah!)